backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -274,19 +274,21 @@ public void loadProperties(List<Endpoint> endpoints) {
                 .filter(e -> e.getType().equals(type))
                 .collect(Collectors.toMap(Endpoint::getId, Function.identity()));
 
-        if (endpointsMap.size() > 0) {
-            String query = "FROM " + typedEndpointClass.getSimpleName() + " WHERE id IN (:ids)";
-            List<T> propList = entityManager.createQuery(query, typedEndpointClass)
-                    .setParameter("ids", endpointsMap.keySet())
-                    .getResultList();
-            for (T props : propList) {
-                if (props != null) {
-                    Endpoint endpoint = endpointsMap.get(props.getId());
-                    endpoint.setProperties(props);
-                    // Todo: NOTIF-429 backward compatibility change - Remove soon.
-                    if (typedEndpointClass.equals(CamelProperties.class)) {
-                        endpoint.getProperties(CamelProperties.class).setSubType(endpoint.getSubType());
-                    }
+        if (endpointsMap.size() <= 0) {
```

Comment:
size can't be below 0. 

It would be fine with a `endpointsMap.size() == 0` to avoid adding confusion.